test/atom/backToTop/index.js

```diff
@@ -17,6 +17,6 @@ describe('atom/backToTop', () => {
     // Example TO BE DELETED!!!!
     // const {getByRole} = render(<AtomButton>HOLA</AtomButton>)
     // expect(getByRole('button')).to.have.text('HOLA')
-    expect(true).to.be.eql(false)
+    expect(true).to.be.eql(true)
```

Comment:
Instead of putting all the tests in green, don't you think it would be better to take an approach like this so that everything that needs to be done is clear?

https://github.mpi-internal.com/scmspain/frontend-ma--uilib-components/pull/1030/files